lib/vault/vault_setup.sh

```diff
@@ -31,11 +31,15 @@ do
   sleep 1
   docker ps --format "{{.Names}}" | grep vault_1
 done
-sleep 5
+sleep 10
```

Comment:
Is there a better way to check if the keys can be grabbed other than this sleep? It might be worth it to just repeat the `docker exec` below until we get a valid response (or if Vault goes down as a failure case), as this might cause problems when trying to troubleshoot CanDIG on slower systems.